dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+| `required`      | Required | Boolean                               | Whether this asset is required.                                                                                                     |
+| `sizes`         | Optional | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.                                             |
+| `aspect_ratios` | Optional | Array[Object]                         | Instead of `sizes`, you can define allowed aspect ratios.  For properties, see [`image.aspect_ratios`](#native-image-aspectratios). |
+
+<a name="native-image-aspectratios" />
+
+###### `adUnit.mediaTypes.native.image.aspect_ratios`
+
+{: .table .table-bordered .table-striped }
+| Name           | Scope    | Type    | Description                                                                                          |
+|----------------+----------+---------+------------------------------------------------------------------------------------------------------|
+| `min_width`    | Optional | Integer | The minimum width required for an image to serve (in pixels).                                        |
+| `ratio_height` | Required | Integer | This, combined with `ratio_width`, determines the required aspect ratio for an image that can serve. |
+| `ratio_width`  | Required | Integer | See above.                                                                                           |
+
+<a name="native-title" />
+
+##### `adUnit.mediaTypes.native.title`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                          |
+|------------+----------+---------+------------------------------------------------------|
+| `required` | Required | Boolean | Whether a title asset is required on this native ad. |
+| `len`      | Optional | Integer | Length of title text, in characters.                 |
+
+<a name="native-sponsoredBy" />
+
+##### `adUnit.mediaTypes.native.sponsoredBy`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                               |
+|------------+----------+---------+-----------------------------------------------------------|
+| `required` | Required | Boolean | Whether a brand name asset is required on this native ad. |
+
+<a name="native-clickUrl" />
+
+##### `adUnit.mediaTypes.native.clickUrl`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                              |
+|------------+----------+---------+----------------------------------------------------------|
+| `required` | Required | Boolean | Whether a click URL asset is required on this native ad. |
+
+##### `adUnit.mediaTypes.native.body`
```

Comment:
The "body" object should have a "len" property.

Reply:
Thanks, updated in be02c24